WinUIGallery/ControlPages/DesignGuidance/AccessibilityScreenReaderPage.xaml

```diff
@@ -460,6 +460,342 @@ AutomationProperties.Name="Contacts"&gt;
 &lt;TextBlock TextWrapping="WrapWholeWords" MaxWidth="400" Margin="8,0,0,0"&gt;This is some demo text.
                 The image on the right is just for decoration and serves no informational purpose.
                 To prevent Narrator or other screen readers from reading out the image, we set the accessibility view to "Raw" which removes it from the content visual tree.&lt;/TextBlock&gt;
+&lt;/StackPanel&gt;
+                </x:String>
+            </core:ControlExample.Xaml>
+        </core:ControlExample>
+
+        <TextBlock
+            Margin="0,20,0,0"
+            Style="{ThemeResource SubtitleTextBlockStyle}"
+            AutomationProperties.HeadingLevel="2"
+            Text="Landmarks and headings" />
+        <RichTextBlock>
+            <Paragraph>
+                <Bold>Landmarks and headings</Bold> indicate, or label, different sections of a user interface, just like visible headings would.
```

Comment:
Should we add for whom it indicates/labels the section since it does not do that on the graphical UI level?

Reply:
Good point!